test/monitoringe2e/helpers.go

```diff
@@ -168,11 +168,11 @@ func VerifyMetrics(caCtx *test.Context, metricQueries []string) error {
 }
 
 func getBearerTokenForPrometheusAccount(caCtx *test.Context) (string, error) {
-	sa, err := caCtx.Clients.Kube.CoreV1().ServiceAccounts("openshift-monitoring").Get(context.Background(), "prometheus-k8s", metav1.GetOptions{})
+	secrets, err := caCtx.Clients.Kube.CoreV1().Secrets("openshift-monitoring").List(context.Background(), metav1.ListOptions{})
```

Comment:
So, OCP 4.11 / monitoring is breaking change?

Reply:
Yes they set `automountServiceAccountToken: false` on the prometheus-k8s sa so you cant get the token by listing the secrets from that sa.